velox/docs/functions/spark/datetime.rst

```diff
@@ -86,10 +86,10 @@ These functions support TIMESTAMP and DATE input types.
     date formatter in lenient mode that is align with Spark legacy date parser behavior or
     `Joda <https://www.joda.org/joda-time/>` date formatter depends on ``spark.legacy_date_formatter`` configuration.
     `Valid patterns for date format
-    <https://spark.apache.org/docs/latest/sql-ref-datetime-pattern.html>`_. Throws exception for
-    invalid ``format``. This function will convert input to milliseconds, and integer overflow is
-    allowed in the conversion, which aligns with Spark. See the below third example where INT64_MAX
-    is used, -1000 milliseconds are produced by INT64_MAX * 1000 due to integer overflow. ::
+    <https://spark.apache.org/docs/latest/sql-ref-datetime-pattern.html>`_. When `Simple` Date Formatter is used,
+    it returns null for invalid ``format``; if not, it throws exception. This function will convert input to
```

Comment:
nit:
When \`Simple\` date formatter is used, null is returned for invalid \``format\``; otherwise, exception is thrown.

Reply:
Thanks. This is clearer.